docker/upload.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+TAG=$(date +'%Y-%m-%d_%H-%M-%S')
```

Comment:
Only core teams can upload new image . Everyone only pull and customize from their own